src/test/Fake.Core.UnitTests/Fake.DotNet.FxCop.fs

```diff
@@ -0,0 +1,323 @@
+module Fake.DotNet.FxCop
+
+open System
+open Fake.Core
+open Fake.DotNet
+open Fake.IO
+open Fake.IO.FileSystemOperators
+open Expecto
+
+let serializingObject = Object()
+
+let testCases =
+    if Environment.isWindows then
+        [ testCase "Test that default arguments are processed as expected" <| fun _ ->
+              let p = FxCop.Params.Create()
+              let dummy = Guid.NewGuid().ToString()
+              let args = FxCop.createArgs p [ dummy ]
+              Expect.isTrue (p.IncludeSummaryReport)
+                  "A field should have non-default value for a bool"
+              Expect.equal args [ "/c"
+                                  "/f:\"" + dummy + "\""
```

Comment:
by the way, have you actually run the code? I feel like this quote might be quoted again later and therefore not work as expected.